resources/assets/js/store/Experience/experienceActions.ts

```diff
@@ -308,6 +312,108 @@ export const deleteExperienceSkill = (
     { id, experienceId, experienceType },
   );
 
+export const BATCH_CREATE_EXPERIENCE_SKILLS_STARTED =
+  "EXPERIENCE SKILL: BATCH CREATE STARTED";
+export const BATCH_CREATE_EXPERIENCE_SKILLS_SUCCEEDED =
+  "EXPERIENCE SKILL: BATCH CREATE SUCCEEDED";
+export const BATCH_CREATE_EXPERIENCE_SKILLS_FAILED =
+  "EXPERIENCE SKILL: BATCH CREATE FAILED";
+
+export type BatchCreateExperienceSkillsAction = AsyncFsaActions<
+  typeof BATCH_CREATE_EXPERIENCE_SKILLS_STARTED,
+  typeof BATCH_CREATE_EXPERIENCE_SKILLS_SUCCEEDED,
+  typeof BATCH_CREATE_EXPERIENCE_SKILLS_FAILED,
+  ExperienceSkill[],
+  {}
+>;
+
+export const batchCreateExperienceSkills = (
+  experienceSkills: ExperienceSkill[],
+): RSAActionTemplate<
+  typeof BATCH_CREATE_EXPERIENCE_SKILLS_STARTED,
+  typeof BATCH_CREATE_EXPERIENCE_SKILLS_SUCCEEDED,
+  typeof BATCH_CREATE_EXPERIENCE_SKILLS_FAILED,
+  ExperienceSkill[],
+  {}
+> =>
+  asyncPost(
+    getBatchCreateExperienceSkillsEndpoint(),
+    experienceSkills,
+    BATCH_CREATE_EXPERIENCE_SKILLS_STARTED,
+    BATCH_CREATE_EXPERIENCE_SKILLS_SUCCEEDED,
+    BATCH_CREATE_EXPERIENCE_SKILLS_FAILED,
+    parseBatchExperienceSkills,
+    {},
+  );
+
+export const BATCH_UPDATE_EXPERIENCE_SKILLS_STARTED =
+  "EXPERIENCE SKILL: BATCH UPDATE STARTED";
+export const BATCH_UPDATE_EXPERIENCE_SKILLS_SUCCEEDED =
+  "EXPERIENCE SKILL: BATCH UPDATE SUCCEEDED";
+export const BATCH_UPDATE_EXPERIENCE_SKILLS_FAILED =
+  "EXPERIENCE SKILL: BATCH UPDATE FAILED";
+
+export type BatchUpdateExperienceSkillsAction = AsyncFsaActions<
+  typeof BATCH_UPDATE_EXPERIENCE_SKILLS_STARTED,
+  typeof BATCH_UPDATE_EXPERIENCE_SKILLS_SUCCEEDED,
+  typeof BATCH_UPDATE_EXPERIENCE_SKILLS_FAILED,
+  ExperienceSkill[],
+  {}
+>;
+
+export const batchUpdateExperienceSkills = (
+  experienceSkills: ExperienceSkill[],
+): RSAActionTemplate<
+  typeof BATCH_UPDATE_EXPERIENCE_SKILLS_STARTED,
+  typeof BATCH_UPDATE_EXPERIENCE_SKILLS_SUCCEEDED,
+  typeof BATCH_UPDATE_EXPERIENCE_SKILLS_FAILED,
+  ExperienceSkill[],
+  {}
+> =>
+  asyncPut(
```

Comment:
I think we should make all the Batch endpoints POST, as mentioned here https://github.com/GCTC-NTGC/TalentCloud/pull/4785/files#r536387663